### ISSUE TITLE
Moving to zero dep

### ISSUE THREAD

**Issue by asim** [micro/micro#1977]
I believe micro needs to become a zero dep system to succeed in the long term. It's clear now people will choose other tools based on their needs for pluggability and infrastructure tooling. Micro can be a distributed service runtime if it provides the core building blocks as a starting point. It should also move to being a single process Daemon.

**Comment by felixd** [comment]
I totally agree. There are to many problems to run it straight on a server with basic components and make it run without issues. 

I am strugling with some "basic problems"
- how to run in "correctly" on production? - tutorial is missing 
- `micro run github.com/micro/services/helloworld` does not run correctly after restarting machine and running `micro server` again. Every microservice needs to be run again using `micro run` command.
- I have made systemd service. It start `micro server` correctly now, but as I have mentioned line above, microservices are not loaded correctly after machine is rebooted. Every microservice has to be run again using `micro run` command. I have tested it using `systemd` and using `micro server` command run manually from bash/terminal. Same issue.
- Why some files are kept in `/tmp/` location and some in `~/.micro`?

Those are just some questions. Going back to work making it run ;)
